cogs/member_stats.py

```diff
@@ -1,9 +1,9 @@
+import discord
 from discord.ext import commands
 from pathlib import Path
 from .validation import is_moderator, is_mod_commands_channel
 import json
 
-
```

Comment:
I believe a newline should be here on purpose, so that we stick to PEP8 more. You might be interested with checking the style through https://github.com/PyCQA/pycodestyle, or setting up a linter on your text editor.